controllers/actions.github.com/resourcebuilder.go

```diff
@@ -19,14 +20,92 @@ const (
 	jitTokenKey = "jitToken"
 )
 
-// labels applied to resources
+// Labels applied to resources
 const (
-	LabelKeyAutoScaleRunnerSetName      = "auto-scaling-runner-set-name"
-	LabelKeyAutoScaleRunnerSetNamespace = "auto-scaling-runner-set-namespace"
+	// Kubernetes labels
+	LabelKeyKubernetesPartOf    = "app.kubernetes.io/part-of"
+	LabelKeyKubernetesComponent = "app.kubernetes.io/component"
+	LabelKeyKubernetesVersion   = "app.kubernetes.io/version"
+
+	// Github labels
+	LabelKeyGithubScaleSetName      = "actions.github.com/scale-set-name"
+	LabelKeyGithubScaleSetNamespace = "actions.github.com/scale-set-namespace"
+	LabelKeyGithubEnterprise        = "actions.github.com/enterprise"
+	LabelKeyGithubOrganization      = "actions.github.com/organization"
+	LabelKeyGithubRepository        = "actions.github.com/repository"
+	LabelKeyGithubRunnerGroupName   = "actions.github.com/runner-group-name"
 )
 
+var commonLabelKeys = [...]string{
+	LabelKeyKubernetesPartOf,
+	LabelKeyKubernetesComponent,
+	LabelKeyKubernetesVersion,
+	LabelKeyGithubScaleSetName,
+	LabelKeyGithubScaleSetNamespace,
+	LabelKeyGithubEnterprise,
+	LabelKeyGithubOrganization,
+	LabelKeyGithubRepository,
+}
+
+const labelValueKubernetesPartOf = "actions-runner-controller"
+
 type resourceBuilder struct{}
 
+func (b *resourceBuilder) newAutoScalingListener(autoscalingRunnerSet *v1alpha1.AutoscalingRunnerSet, ephemeralRunnerSet *v1alpha1.EphemeralRunnerSet, namespace, image string, imagePullSecrets []corev1.LocalObjectReference) (*v1alpha1.AutoscalingListener, error) {
```

Comment:
do you mind moving the method back to the line it was on to reduce the diff?

Reply:
Sure, I wanted to move stuff around so it is easier to find, but that might be something for another PR to make this one easier to review :relaxed: